<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>beidouData</class>
 <widget class="QWidget" name="beidouData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>数据接入</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>设备地址</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ip_lineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>设备端口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="port_lineEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>断线重连</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="reconnect_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>定位信息（$DWXX）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>用户地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="user_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>长度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="length_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>查询地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>经度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lon_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>类别    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="type_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>纬度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lat_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>密钥    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>高度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="height_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>精度    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="jd_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>小时</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hour_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>紧急定位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="urgency_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>分钟</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="min_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>多值解  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="more_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>秒  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sec_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>高程类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="heighttype_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>0.01秒</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mes_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>数据生成</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>北斗数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="beidou_lineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>数据输出</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>输出端口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="output_lineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>数据标识</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="flag_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>保存配置</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
